modules/manage/pages/topic-iceberg-integration.adoc

```diff
@@ -0,0 +1,117 @@
+
+== Prerequisites
+
+include::shared:partial$enterprise-license.adoc[]
+
+. To check if you already have a license key applied to your cluster:
++
+[,bash]
+----
+rpk cluster license info
+----
+
+. Install `rpk`.
+
+== Limitations
+
+* You can only enable the Iceberg integration when creating a new Redpanda topic. 
+* It is not possible to append or backfill data from Redpanda topics to an existing Iceberg table.
+* JSON schemas are not currently supported.
+* You can only use one schema per topic. Schema versioning as well as upcasting (where a message is cast into its more generic data type) are not supported. 
+
+== Architecture 
+
+Apache Iceberg is an open source format specification for defining structured tables in a glossterm:data lake[]. The table format enables you to easily and quickly manage, query, and process huge amounts of structured and unstructured data, like you would manage and run SQL queries against relational data in a database or data warehouse. The open format allows many different languages, tools, and applications to process the same data in a consistent way, so you can avoid vendor lock-in. This data management system is also referred to as a _data lakehouse_.
+
+In the Iceberg specification, tables comprise the following:
+
+* Data layer
+** Data files: Store the actual data. The Redpanda integration currently supports the Parquet file format. Parquet files are column-based and suitable for analytical workloads at scale, and come with compression capabilities that optimize files for object storage. 
+* Metadata layer
++
+--
+** Manifest files: Track data files and contain metadata about those files, such as record count, partition membership, and file paths.
+** Manifest list: Tracks all the manifest files belonging to a table, including file paths and upper and lower bounds for partition fields. 
+** Metadata file: Stores metadata about the table, including its schema, partition information, and snapshots. Whenever a change is made to the table, a new metadata file is created and becomes the latest version of the metadata in the catalog.
+--
++
+In the Redpanda Iceberg integration, the manifest files are in JSON format.
+* Catalog: Contains the current metadata pointer for the table. Clients reading and writing data to the table see the same version of the current state of the table. Redpanda creates the catalog in JSON format.
+
+When you enable the Iceberg integration for a Redpanda topic, Redpanda brokers store streaming data in the Iceberg-compatible format in Parquet files in object storage, in addition to writing Kafka messages stored in segment files on disk. Storing the streaming data in Iceberg tables in the cloud allows you to derive real-time insights through business intelligence and machine learning tools that work with Iceberg. 
+
```

Comment:
Should we use the same [graphic](https://www.redpanda.com/blog/apache-iceberg-topics-streaming-data) from the blog post for this section? Matt also used one in his SE enablement [slide deck](https://docs.google.com/presentation/d/1vLI46jHyjO6uaqe7dt2SFzUjDdDHmrT52S6OnZuRJWY/edit?usp=sharing) (slide 8) that might be a fit, but it probably needs to be edited as we do not mean to imply that the Iceberg tables can also be accessed from the read cache.

Reply:
I think the graphic from the blog post is actually really good.

Reply:
I've added the image from the blog post for now, but will request a new version for docs from the design team.